src/odemis/gui/cont/tabs.py

```diff
@@ -1352,19 +1352,24 @@ def _on_switch_btn(self, evt):
         # Note: s axis can only be moved when near engaged pos. So:
         #  * when parking/referencing => move s first, then l
         #  * when engaging => move l first, then s
+        # Some systems are special, default is overridden with MD_AXES_ORDER_REF
         mirror = self.tab_data_model.main.mirror
+        axes_order = mirror.getMetadata().get(model.MD_AXES_ORDER_REF, ("s", "l"))
+        if set(axes_order) != {"s", "l"}:
+            logging.warning("Axes order of mirror is %s, while should have s and l axes",
```

Comment:
while it should have order s and then l?

Reply:
No, here the order doesn't matter. What I'm checking is that the user didn't enter weird axes names like `"s "`, or `"L"`.